src/class-wpml-elementor-integration-factory.php

```diff
@@ -20,7 +20,8 @@ public function create() {
 				'WPML_Elementor_Adjust_Global_Widget_ID_Factory',
 				'WPML_PB_Elementor_Handle_Custom_Fields_Factory',
 				'WPML_Elementor_Media_Hooks_Factory',
-				'WPML_Elementor_WooCommerce_Hooks_Factory'
+				'WPML_Elementor_WooCommerce_Hooks_Factory',
+				'\WPML\PB\Elementor\LanguageSwitcher\Factory'
```

Comment:
@amElnagdy We no longer need factory classes for simple hooks now. You can change this to `'\WPML\PB\Elementor\LanguageSwitcher`. I'll add comments in other places about what else to change.